articles/container-instances/container-instances-managed-identity.md

```diff
@@ -383,6 +375,29 @@ identity:
   userAssignedIdentities:
    {'myResourceID1':{}}
 ```
+## Managed Identity on Windows containers
+
+Managed Identity on Windows container groups works differently than Linux container groups. For Windows containers, metadata server (169.254.169.254) is not available for getting the AAD token. Customers can follow a different pattern to get the access token in Windows containers. The pattern involves sending token request to the IDENTITY_ENDPOINT along with additional information such as principal id and secret as shown below. The IDENTITY_ENDPOINT and IDENTITY_HEADER are injected as environmental variable in your container.
```

Comment:
"AAD" is not a term that we can use on public facing docs.  Perhaps this can be changed to Microsoft Entra ID ?